eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -254,7 +265,7 @@ stages:
                 runOnce:
                   deploy:
                     steps:
-                      - checkout: self
+                      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
```

Comment:
Is this going to work here? For the java version updates below they could potentially change files all over the repo.

Reply:
You're right, I probably need to glob xml for `update_versions.py`. I haven't been able to get this far in release testing because of the dependency issues with `template` mentioned in the description.